### DISCUSSION TITLE
xboard.dev - Using pip-tools to manage project dependencies in Python

### DISCUSSION THREAD

**Discussion by giscus** [xboard/quarto-blog#11] [Announcements]
# xboard.dev - Using pip-tools to manage project dependencies in Python

Using pip-tools to manage project dependencies in Python. How to prevents conflicts between versions of packages and ensure your project dependencies are isolated and you have a reproducible development environment.

https://www.xboard.dev/posts/2023_07_20_using_pip_tools_to_manage_python_project_dependencies/

**Comment by thsant** [comment]
Very, very useful post, thank you: my venvs usually become a mess.
